onnxruntime/python/tools/kernel_explorer/kernels/rocm/gemm_fast_gelu_ck.cu

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is there a way to preserve the commit history of this file, instead of adding a new file?

Reply:
this commit(https://github.com/microsoft/onnxruntime/pull/13759/commits/43f50cb898f089562a1b8da3b5c2ea62dbf38c32) change the structure of gemmfastgelu code before adding ck. we can split this commit as single pr if we need.